### PR TITLE
Feature/map 155/create frontend part edit location

### PR DESCRIPTION
**Pull request by Yojuks**

### CODE REVIEW

**Review by obozhko-dev — APPROVED** [review]

File: frontend/src/components/PointPopup/PointPopup.tsx [hunk 1]
```diff
@@ -10,11 +10,20 @@ import {
...
 import { CardComponent } from './СardComponent/CardComponent';
 import { IconsComponent } from './IconsComponent/IconsComponent';
 import { StyledPopupButtonsWrapper } from '../design/StyledPopupButtonsWrapper';
 
+import { LocationForm } from '../../../types';
+import EditLocation from '../EditLocation/EditLocation';
+
+// import { useDispatch } from 'react-redux';
+// import { yupResolver } from '@hookform/resolvers/yup';
+// import { EditProfileSchema } from 'utils/validation';
```
Comment (vasylneshchadimov) [inline frontend/src/components/PointPopup/PointPopup.tsx]: please dellete if you do not use them

**Review by vasylneshchadimov — COMMENTED** [review]
bag

After I click button save, i see snack bar with message 'Location create'. As for me, will better if message will-'Location Edited"

**Review by vasylneshchadimov — COMMENTED** [review]
In my opinian, will better if user after  click button save, and snack bar show message, popup will look without  edit buttons(save, cancel) and with new data

**Comment by Yojuks** [comment]
> In my opinian, will better if user after click button save, and snack bar show message, popup will look without edit buttons(save, cancel) and with new data

**Comment by Yojuks** [comment]
an error occurred and I accidentally closed this pr

File: frontend/src/components/PointPopup/PointPopup.tsx (hunk 1 above)
Comment (Yojuks) [reply frontend/src/components/PointPopup/PointPopup.tsx]: i added validation and delete not used commented code

**Comment by Yojuks** [comment]
> In my opinian, will better if user after click button save, and snack bar show message, popup will look without edit buttons(save, cancel) and with new data

i fixed this part

**Review by ghost — APPROVED** [review]

**Pull request merged**